src/integration/pages/homePage/portuguese/canonical.test.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * @service portuguese
+ * @pathname /portuguese
+ */
+
+import { data as pageData } from '../../../../../data/portuguese/homePage/index.json';
+
+describe('Canonical', () => {
+  let portraitVideoCurations;
+
+  beforeAll(() => {
+    portraitVideoCurations = pageData.curations.filter(
+      curation =>
+        curation.portraitVideo && Array.isArray(curation.portraitVideo.items),
+    );
+  });
```

Comment:
I don't even think you need to do this in a beforeAll: 
```suggestion
    const portraitVideoCurations = pageData.curations.filter(
      curation =>
        curation.portraitVideo && Array.isArray(curation.portraitVideo.items),
    );
```

Reply:
I have taken it out of the beforeAll 👍